aiida_quantumespresso/parsers/parse_xml/parse.py

```diff
@@ -138,8 +138,11 @@ def parse_xml_post_6_2(xml):
 
     lsda = inputs.get('spin', {}).get('lsda', False)
     spin_orbit_calculation = inputs.get('spin', {}).get('spinorbit', False)
-    non_colinear_calculation = outputs['magnetization']['noncolin']
-    do_magnetization = outputs['magnetization'].get('do_magnetization', False)
+    non_colinear_calculation = False
+    do_magnetization = False
+    if 'magnetization' in outputs:
+        non_colinear_calculation = outputs['magnetization']['noncolin']
+        do_magnetization = outputs['magnetization'].get('do_magnetization', False)
```

Comment:
Think the following is easier to read
```suggestion
    non_colinear_calculation = outputs.get('magnetization', {}).get('noncolin', False)
    do_magnetization = outputs.get('magnetization', {}).get('do_magnetization', False)
```